.circleci/deleted_bazel_packages.txt

```diff
@@ -0,0 +1 @@
+tools/clang_json
```

Comment:
@oferb it does not help much because `--proto_compiler` and `--proto_toolchain_for_java` options are not available in `query` [is this an issue to create in `google/protobuf`?] and when we don't specify them, `bazel` will implicitly make a dependency i.e. between each `proto_library` rule and `@com_google_protobuf//:protoc`